src/cmd/cmdfuel.rb.in

```diff
@@ -422,6 +440,12 @@ class Cmd
                                options['owner'])
           exit(-1)
         end
+      when 'update'
+        Importer.extern 'int update(const char *, const char *)'
+        if not Importer.update(options['onlymodels'],
+                               options['onlyworlds'])
```

Comment:
Add in header support using the `options['header']`.

Reply:
Added in bff466c